core/src/test/java/org/elasticsearch/snapshots/mockstore/MockRepository.java

```diff
@@ -116,6 +116,7 @@ public MockRepository(RepositoryMetaData metadata, Environment environment,
         blockOnInitialization = metadata.settings().getAsBoolean("block_on_init", false);
         randomPrefix = metadata.settings().get("random", "default");
         waitAfterUnblock = metadata.settings().getAsLong("wait_after_unblock", 0L);
+        nonAtomicMove = metadata.settings().getAsBoolean("non_atomic_move", false);
```

Comment:
I think negatively named settings are somewhat difficult to reason about. For example, I think it's much easier to understand that `atomicMove = true` is than `nonAtomicMove = false`.